docs/development/quickstart.md

```diff
@@ -166,10 +166,7 @@ poetry run pytest tests --runslow -n auto
 
 ## Hacking the Web client (front)
 
-In addition to the [shared requirements](#shared-requirements), you need to install [`Node 18.12.0`](https://nodejs.org/en/download/releases).
-
-> You can use [`nvm`](https://github.com/nvm-sh/nvm) to manage multiple node version
-> or [`nvm-windows`](https://github.com/coreybutler/nvm-windows) for `Windows`
+In addition to the [shared requirements](#shared-requirements), for working with the web clients you need to:
```

Comment:
```suggestion
In addition to the [shared requirements](#shared-requirements), for working with the web client you need to:
```

Reply:
For me we have multiple clients `web`, `electron`, `android` & `ios` :thinking:

Reply:
So "working with the ~web~ clients"? I only suggested that to be coherent with the title (which should probably also be changed to "Hacking the ~web~ client").

IMHO even if the clients share the same core it would seem weird to me to refer to the "android web client", I will prefer "clients" as in "web client", "android client", ...

Reply:
```suggestion
In addition to the [shared requirements](#shared-requirements), for working with the clients you need to:
```

Reply:
Currently the section `Hacking the web client` provide steps for the web & electron, maybe I should split that ?

Reply:
idk, I guess it's also fine as it is... we can split that later when we add android/ios
So as you wish :)

Reply:
I will prepare the terrain for it then :)

Reply:
I've split the section